tklinenums/tklinenums.py

```diff
@@ -114,9 +116,34 @@ def redraw(self, *_) -> None:
         last_line = int(
             self.textwidget.index(f"@0,{self.textwidget.winfo_height()}").split(".")[0]
         )
-
+        
+        # Set the default loop range
+        _range = last_line + 1
+
+        # This runs if user is using a str as font. Example: "monospace 11"
+        # This is not recommended, since font should be a instance of Font class of Tkinter or Customtkinter
+        # But, because projects that use TkLineNums probably use str type fonts, this fix the situation
+        if type(self.textwidget.cget("font")) == str:
+            cur_font = self.textwidget.cget("font").split(" ")
+            _font = Font(family=cur_font[0], size=cur_font[1])
```

Comment:
may you please show me what is your textwidget font?
This error was worring me, because if user's textbox font is not str or tkinter.Font or even customtkinter.Font, it will happen
I was thinking in using a raise error in this case, tell user to use their font as a instance of tkinter Font class

Reply:
I'm just running the code in the `if __name__ == "__main__"

```python
if __name__ == "__main__":
    from tkinter import Tk
    from tkinter.ttk import Style

    root = Tk()

    style = Style()
    style.configure("TkLineNumbers", foreground="#2197db", background="#ffffff")

    text = Text(root)
    text.pack(side="right")

    for i in range(50):
        text.insert("end", f"Line {i+1}\n")

    def ttk_theme_colors() -> tuple[str, str]:
        fg: str = style.lookup("TkLineNumbers", "foreground", default="black")
        bg: str = style.lookup("TkLineNumbers", "background", default="white")
        return (fg, bg)

    linenums = TkLineNumbers(root, text, colors=ttk_theme_colors)
    linenums.pack(fill="y", side="left", expand=True)

    text.bind("<<Modified>>", lambda _: linenums.redraw())
    text.config(font=("Courier New bold", 15))

    root.mainloop()
```

Reply:
`print(self.textwidget.cget("font"))` gives `TkFixedFont`

Reply:
>     text.config(font=("Courier New bold", 15))

I think the error happened because font is a tuple, and in this case, it deal as a string. I will fix that!
If you use `font="monospace 11"`, for example, it should work fine.
also, `font=tk.Font(family="Coutier New bold", size=15)` should work too!